pkg/querybackend/client/client.go

```diff
@@ -63,6 +63,6 @@ const grpcServiceConfig = `{
               "RESOURCE_EXHAUSTED"
             ]
         },
-        "timeout": "20s"
```

Comment:
I am not quite sure what other timeouts on our whole stack are, but I agree 30s than 20s is better, maybe we could make this a CLI flag?